src/facade/dragonfly_connection.cc

```diff
@@ -885,6 +886,8 @@ void Connection::ConnectionFlow(FiberSocketBase* peer) {
   // After the client disconnected.
   cc_->conn_closing = true;  // Signal dispatch to close.
   evc_.notify();
+  queue_backpressure_->done = true;
```

Comment:
so I suggest dchecking on the value of subscriber_bytes first. it should be 0 when connection is shutting down and if it's not we have other bugs.